### PR TITLE
Make UI Tests work with new Build Variants

### PR DESCRIPTION
**Pull request by npark-mozilla**
- Move all UI Tests into AndroidTest folder
- For Webview only tests, use Assume() to ignore tests that are either Klar or Geckoview
- For tests that run on both webview and geckoview, change to geckoview when the app is Klar
- All Debug builds are currently using webview

### CODE REVIEW

**Comment by npark-mozilla** [comment]
Handles #2973

File: app/src/androidTest/java/org/mozilla/focus/activity/AccessAboutAndYourRightsPagesTest.java [hunk 1]
```diff
@@ -73,7 +79,7 @@ public void tearDown() {
     }
 
     @Test
-    public void AccessAboutAndYourRightsPagesTest() {
+    public void AboutAndYourRightsPagesTest() {
```
Comment (sputnikci) [inline app/src/androidTest/java/org/mozilla/focus/activity/AccessAboutAndYourRightsPagesTest.java]: \[PMD\] INFO: The method 'AboutAndYourRightsPagesTest\(\)' has a NCSS line count of 23\.

File: app/src/androidTest/java/org/mozilla/focus/screenshots/FirstRunScreenshots.java [hunk 2]
```diff
@@ -26,7 +29,20 @@
 @RunWith(AndroidJUnit4.class)
 public class FirstRunScreenshots extends ScreenshotTest {
     @Rule
-    public ActivityTestRule<MainActivity> mActivityTestRule = new MainActivityFirstrunTestRule(true);
+    public ActivityTestRule<MainActivity> mActivityTestRule = new MainActivityFirstrunTestRule(true){
```
Comment (sputnikci) [inline app/src/androidTest/java/org/mozilla/focus/screenshots/FirstRunScreenshots.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

File: app/src/androidTest/java/org/mozilla/focus/screenshots/ScreenshotTest.java [hunk 3]
```diff
@@ -30,6 +34,22 @@
 
     UiDevice device;
 
+    @Rule
+    public ActivityTestRule<MainActivity> mActivityTestRule = new MainActivityFirstrunTestRule(false){
```
Comment (sputnikci) [inline app/src/androidTest/java/org/mozilla/focus/screenshots/ScreenshotTest.java]: \[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

**Pull request merged**